### PR TITLE
Release Notes Overview update

### PR DESCRIPTION
**Pull request by jeff2d2**
Fixes department-of-veterans-affairs/vets-contrib#3152.
Revised verbiage for Overview on the Release Notes page.

### CODE REVIEW

File: src/containers/releaseNotes/ReleaseNotes.tsx [hunk 1]
```diff
@@ -64,7 +64,19 @@ function renderOverview(routeProps: any, props: any) {
...
 export class ReleaseNotes extends React.Component<RouteComponentProps<IApiNameParam>, {}> {
   private overviewProps = {
-    description: 'The goal of the Release Notes section is to inform developers of updates and changes to the VA APIs. This section of the developer portal is new and will expand as release notes for new APIs become available.', 
+    description:
+      <div>
+        <p>
+          The VA Lighthouse product teams periodically update these APIs in order to deliver new features and repair defects. Occasionally we need to make "breaking changes" that require developers to modify their existing applications.
+        </p>
+        <p>
+          We recommend that developers periodically check this list for announcements of breaking changes and added features. Changes will also be announced via direct email whenever possible to addresses used to obtain developer keys for each API. Please <a href="https://developer.va.gov/support/contact-us">contact us</a> with any questions, or to request support.
```
Comment (rtravitz) [inline src/containers/releaseNotes/ReleaseNotes.tsx]: The comma splice before "or request support".

File: src/containers/releaseNotes/ReleaseNotes.tsx [hunk 2]
```diff
@@ -64,7 +64,19 @@ function renderOverview(routeProps: any, props: any) {
 
 export class ReleaseNotes extends React.Component<RouteComponentProps<IApiNameParam>, {}> {
   private overviewProps = {
-    description: 'The goal of the Release Notes section is to inform developers of updates and changes to the VA APIs. This section of the developer portal is new and will expand as release notes for new APIs become available.', 
+    description:
+      <div>
+        <p>
+          The VA Lighthouse product teams periodically update these APIs in order to deliver new features and repair defects. Occasionally we need to make "breaking changes" that require developers to modify their existing applications.
```
[the remark this answers is not in the document]
Comment (nfasulo) [reply src/containers/releaseNotes/ReleaseNotes.tsx]: :+1: This gave me pause as well.

[the remark this answers is not in the document]
Comment (jeff2d2) [reply src/containers/releaseNotes/ReleaseNotes.tsx]: I like that better. Fixed!

**Comment by Eallan919** [comment]
@rtravitz for review :)

**Pull request merged**